src/Catalog/PackageCatalogItem.cs

```diff
@@ -12,25 +12,27 @@ namespace NuGet.Services.Metadata.Catalog
 {
     public class PackageCatalogItem : AppendOnlyCatalogItem
     {
-        private readonly NupkgMetadata _nupkgMetadata;
-        private readonly DateTime? _createdDate;
-        private readonly DateTime? _lastEditedDate;
-        private readonly DateTime? _publishedDate;
         private string _id;
         private string _fullVersion;
         private string _normalizedVersion;
 
+        // These properties are public only to facilitate testing.
+        public NupkgMetadata NupkgMetadata { get; }
```

Comment:
Make them `internal` then maybe, and add test assembly in `InternalsVisibleTo`?

Reply:
I didn't do it because it's not worth the effort now.